src/app/src/test/kotlin/org/imperial/mrc/hint/integration/DiseaseTests.kt

```diff
@@ -19,24 +21,20 @@ class DiseaseTests(@Autowired val testRestTemplate: TestRestTemplate) {
         File(tmpUploadDirectory).deleteRecursively()
     }
 
-    @Disabled
     @Test
     fun `can upload survey file`() {
-
         val postEntity = createTestHttpEntity()
         val entity = testRestTemplate.postForEntity<String>("/disease/survey/", postEntity)
         assertThat(entity.statusCode).isEqualTo(HttpStatus.BAD_REQUEST)
-        assertThat(entity.body!!).isEqualTo("{\"status\": \"failure\", \"errors\": [], \"data\": {}}")
+        JSONValidator().validateError(entity.body!!, "INVALID_FILE")
```

Comment:
Note hintr is returning 400 for all files at the moment, next step will be to get it to return successful validation responses and send it some fake but valid data.